Source/gravity/gravity_sources_nd.F90

```diff
@@ -77,12 +67,6 @@ subroutine ca_gsrc(lo, hi, &
     src(:) = ZERO
     snew(:) = ZERO
 
-    ! For constant gravity, we can just initialize the gravitational acceleration here.
-
-#ifndef SELF_GRAVITY
-    Sr(dim) = const_grav
```

Comment:
why is this no longer needed?

Reply:
It was never needed. We fill the StateData for g when we're using ConstantGrav. That code was put in there so the version without SELF_GRAVITY would compile, but it was not necessary for correct results.